/jds/bin/diff -uprN speex-1.2rc1.old/src/speexdec.1 speex-1.2rc1/src/speexdec.1
--- speex-1.2rc1.old/src/speexdec.1	2009-02-19 18:34:26.613531000 +0000
+++ speex-1.2rc1/src/speexdec.1	2009-02-19 18:41:46.663317000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.29.
 .TH SPEEXDEC "1" "September 2003" "speexdec version 1.1" "User Commands"
 .SH NAME
@@ -74,5 +75,21 @@ Deprecated, use \fB\-\-no\-enh\fR instea
 More information is available from the Speex site: http://www.speex.org
 .PP
 Please report bugs to the mailing list `speex-dev@xiph.org'.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWspeex
+Interface stability^Volatile
+.TE
+.sp
 .SH COPYRIGHT
 Copyright \(co 2002 Jean-Marc Valin
+...\" LSARC 2005/734 GNOME For Nevada
/jds/bin/diff -uprN speex-1.2rc1.old/src/speexenc.1 speex-1.2rc1/src/speexenc.1
--- speex-1.2rc1.old/src/speexenc.1	2009-02-19 18:34:26.612455000 +0000
+++ speex-1.2rc1/src/speexenc.1	2009-02-19 18:41:50.389731000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.29.
 .TH SPEEXENC "1" "September 2003" "speexenc version 1.1" "User Commands"
 .SH NAME
@@ -101,5 +102,21 @@ Default raw PCM input is 16-bit, little-
 More information is available from the Speex site: http://www.speex.org
 .PP
 Please report bugs to the mailing list `speex-dev@xiph.org'.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWspeex
+Interface stability^Volatile
+.TE
+.sp
 .SH COPYRIGHT
 Copyright \(co 2002 Jean-Marc Valin
+...\" LSARC 2005/734 GNOME For Nevada
